docs/usage/faq.md

```diff
@@ -36,6 +36,26 @@ The maintainers don't follow any release schedule or release cadence.
 This means the Mend Renovate App can lag a few hours to a week behind the open source version.
 Major releases of Renovate are held back until the maintainers are reasonably certain it works for most users.
 
+## How can I see which version the Mend Renovate app is using?
+
+Follow these steps to see which version the Mend Renovate app is on:
+
+1. Go to the [Mend Developer Portal](https://developer.mend.io/)
+1. Sign in to the Renovate app with your GitHub or Bitbucket account
+1. Select the app you would like to log in with: Forking Renovate or Renovate
+1. Select your organization
+1. Select a installed repository
+1. Select a job from the _Recent jobs_ overview
+1. Select the _Info_ Log Level from the dropdown menu
+1. You should see something like this:
+
+   ```
+   INFO: Repository started
+   {
+     "renovateVersion": "37.326.0"
+   }
+   ```
+
```

Comment:
> Also, a possibly quicker alternative would be to select through to the logs via any existing Renovate-authored PR in the desired repo. The link is at the end like, `This PR has been generated by Mend Renovate. View repository job log here.` But maybe that's not particularly worth mentioning due to its indirectness, even if quicker.

I think we can mention it as a tip, but keep the instructions "starting from zero". The link in Renovate's PRs is more of a shortcut thing. :smile: 

```suggestion
<!-- prettier-ignore -->
!!! tip
    The PRs that Renovate creates have a link to the "repository job log" in the footer of the PR body text.

```